src/raftstore/store/fsm/peer.rs

```diff
@@ -468,6 +471,10 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
         }
     }
 
+    fn on_snap_res(&mut self, term: u64) {
+        self.fsm.peer.mut_store().set_applied_term(term);
```

Comment:
Please add more comments to explain why not need to `set_applied_state` here.

Reply:
> apply_state will be applied on applying next raft log.
It's for the next raft log. However for the snapshot, we need also update `PeerStorage::apply_state`. And I think it's done in `PeerStorage::post_ready`. Could you comment it here if you also agree with it?